src/DataLayer/utils/AfmUtils.ts

```diff
@@ -321,9 +323,14 @@ export function appendFilters(
         dateFilters.push(afmDateFilter);
     }
 
-    const afmAttributeFilters = afmFilters.filter(filter => !AFM.isDateFilter(filter));
+    const afmNonDateFilters = afmFilters.filter(filter => !AFM.isDateFilter(filter));
```

Comment:
the structure of this function is terrible/unreadable. I would appreciate some improvement but if you don't have time for that we can merge as this PR doesn't make it any worse.

Reply:
It is not great, I agree with that.
However, I would leave it for now. I would rather work on the next task and then help with sprint goals.